### DISCUSSION TITLE
Error while updating to Laravel 11

### DISCUSSION THREAD

**Discussion by voroninadm** [laravel/framework#50885] [Q&A]
"require": {
        "php": "^8.2",
        "guzzlehttp/guzzle": "^7.2",
        "inertiajs/inertia-laravel": "^1.0",
        "laravel/framework": "^11.0",
        "laravel/reverb": "@beta",
        "laravel/sanctum": "^4.0",
        "laravel/tinker": "^2.9",
        "maatwebsite/excel": "^3.1",
        "spatie/laravel-backup": "^8.6",
        "spatie/laravel-permission": "^5.11",
        "tightenco/ziggy": "^1.0"
    },
    "require-dev": {
        "barryvdh/laravel-debugbar": "^3.11",
        "fakerphp/faker": "^1.9.1",
        "laravel-lang/common": "^3.1",
        "laravel/breeze": "^1.21",
        "laravel/pint": "^1.0",
        "laravel/sail": "^1.18",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^7.0",
        "phpunit/phpunit": "^10.1",
        "spatie/laravel-ignition": "^2.0"
    },
    
      Problem 1
    - illuminate/database[v7.0.0, ..., v7.28.4] require php ^7.2.5 -> your php version (8.2.15) does not satisfy that requirement.
    - illuminate/database[v8.0.0, ..., v8.11.2] require php ^7.3 -> your php version (8.2.15) does not satisfy that requirement.
    - Root composer.json requires inertiajs/inertia-laravel ^1.0 -> satisfiable by inertiajs/inertia-laravel[v1.0.0].
    - Root composer.json requires spatie/laravel-permission ^5.11 -> satisfiable by spatie/laravel-permission[5.11.0, 5.11.1].
    - Conclusion: don't install laravel/framework v11.0.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v11.0.3 (conflict analysis result)
    - Conclusion: don't install laravel/framework v11.0.4 (conflict analysis result)
    - Conclusion: don't install laravel/framework v11.0.5 (conflict analysis result)
    - Conclusion: don't install laravel/framework v11.0.6 (conflict analysis result)
    - Conclusion: don't install laravel/framework v11.0.7 (conflict analysis result)
    - Conclusion: don't install laravel/framework v11.0.8 (conflict analysis result)
    - Conclusion: don't install laravel/framework v11.1.0 (conflict analysis result)
    - Conclusion: don't install laravel/framework v11.1.1 (conflict analysis result)
    - spatie/laravel-permission[5.11.0, ..., 5.11.1] require illuminate/database ^7.0|^8.0|^9.0|^10.0 -> satisfiable by illuminate/database[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, 
..., v9.52.16, v10.0.0, ..., v10.48.4].
    - Only one of these can be installed: illuminate/database[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.4, v11.0.0, ..., v11.1.1], laravel/framewo
rk[v11.0.0, ..., v11.1.1]. laravel/framework replaces illuminate/database and thus cannot coexist with it.
    - inertiajs/inertia-laravel v1.0.0 requires laravel/framework ^8.74|^9.0|^10.0|^11.0 -> satisfiable by laravel/framework[v11.0.0, ..., v11.1.1].
    - Conclusion: don't install laravel/framework v11.0.1 (conflict analysis result)

Hello everybody! Help please, how to update?  I've tryed to add -W but it's not work

**Comment by voroninadm** [comment]
ok

**Comment by voroninadm** [comment]
ok

**Comment by bradley-varol** [comment]
I have this same issue. It is because my Laravel Nova version is out of date.

I'm not sure what to do because my license has expired. Can I seriously not upgrade my Laravel project of an expired Nova license?
